sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/OtlpConfigUtil.java

```diff
@@ -171,6 +171,30 @@ private static URL validateEndpoint(@Nullable String endpoint, boolean allowPath
     return endpointUrl;
   }
 
+  private static byte[] readPemByConfig(ConfigProperties config, String prefix, String dataType, String suffix ) {
+    String propertyToRead = prefix + "." + dataType + "." + suffix;
+    String filePath = config.getString(propertyToRead);
+    if (filePath == null) {
+      propertyToRead = prefix + "." + suffix;
+      filePath = config.getString(propertyToRead);
+    }
+    if (filePath != null) {
```

Comment:
How about exiting early if `filePath == null` here?
```
if (filePath == null) {
 return null;
}
...
```

Reply:
Existing code tries to load data according to specific property (log, metric, span) and if not existing then generic one without datatype is used. Exiting early would change this logic and not use generic property. 
Maybe I didn't understood this proposal but for now I will keep this part as it is. Could you please elaborate a little more?

Reply:
If you exit early with, the the code inside this if block doesn't need to be indented:
```
if (filePath == null) {
  return null;
}
Path path = Paths.get(filePath);
if (!Files.exists(path)) {
  throw new ConfigurationException(
      "Invalid file: " + path + " (configured in property: " + propertyToRead + ")");
}
try {
  return Files.readAllBytes(path);
} catch (IOException e) {
  throw new ConfigurationException(
      "Error reading content of file (" + path + ") configured in " + propertyToRead, e);
}
```

Reply:
ok leaving early